app/publish/publish.go

```diff
@@ -107,9 +108,12 @@ func (h Handler) Handle(w http.ResponseWriter, r *http.Request) {
 		op := metrics.StartOperation(opName, "remove_file")
 		defer op.End()
 
-		if err := os.Remove(f.Name()); err != nil {
-			monitor.ErrorToSentry(err, map[string]string{"file_path": f.Name()})
+		newFile, err := os.Create(f.Name())
+		if err != nil {
+			log.Error(err)
 		}
+		defer os.RemoveAll(filepath.Dir(newFile.Name()))
```

Comment:
It's necessary to log a potential error returned from `RemoveAll`, check how it's already being done in the file removal procedure that you deleted a few lines above.

Reply:
Fixed creating the file again. I also added back the logging that was being done previously.

Reply:
The cleanup block is already running deferred.